openconcept/analysis/openaerostruct/aerostructural.py

```diff
@@ -1292,8 +1291,8 @@ def setup(self):
 
 class OASAerostructDragPolarExact(om.Group):
     """
-                          WARNING WARNING WARNING
-    ----------------------------------------------------------------------
+    *WARNING WARNING WARNING*
```

Comment:
Should use a numpydoc warning section instead (after docstrings) for better rendering. Same applies below